rclcpp/include/rclcpp/node.hpp

```diff
@@ -146,6 +148,24 @@ class Node
     FunctorT callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
+  const std::vector<rcl_interfaces::SetParametersResult> set_parameters(
```

Comment:
Why return a `const non-reference` here? If it can't be a reference what is the purpose of `const` then?